rfd/0080/convergence-service-draft.md

```diff
@@ -0,0 +1,49 @@
+# Mariposa Convergence Service
+
+The Mariposa (RFD36) convergence service will be responsible for insuring service goal state is met. It accomplishes this by polling the service and project APIs, the VM API, and by monitoring Changefeed. When a divergence between goal state and actual state is detected, the convergence service will interact with VM API to resolve the discrepancy. 
```

Comment:
> the convergence service will interact with VM API to resolve the discrepancy

Direct interaction here means that limits and other controls set for CloudAPI will not be applied to this service. That already applies to Docker API interactions, which have created just enough inconsistency to make us think about it as a problem.